test/IECoreScene/MeshAlgoDistributePointsTest.py

```diff
@@ -72,6 +72,16 @@ def testSimple( self ) :
 		p = IECoreScene.MeshAlgo.distributePoints( mesh = m, density = 100 )
 		self.pointTest( m, p, 100 )
 
+
+	def testRaisesExceptionIfNoUVs( self ) :
+
+		m = IECore.Reader.create( "test/IECore/data/cobFiles/pCubeShape1.cob" ).read()
+		if 'uv' in m:
```

Comment:
Presumably we do expect `uv` to be in m? If so, let's not mask a potential loading problem with this defensive `if`...